### PR TITLE
Make content hover text use full possible width

### LINKED ISSUE #186318: Hover Content Does Not Use Full Width
Testing #185751

Code_-_Insiders_W34xFKRAkD

Video:

Code_-_Insiders_2L8MPeZ6ox

### PR DESCRIPTION
**Pull request by aiday-mar**
Fixes https://github.com/microsoft/vscode/issues/186318

The max width of the text within the content hover is changed so that the content hover text uses the full possible width. The new behavior can be seen in the following gif:

https://github.com/microsoft/vscode/assets/61460952/443b83aa-be90-4f48-8067-8e60dfcbd239

### CODE REVIEW

File: src/vs/editor/contrib/hover/browser/contentHover.ts [hunk 1]
```diff
@@ -538,12 +538,32 @@ export class ContentHoverWidget extends ResizableContentWidget {
...
+		container.style.maxWidth = transformedWidth;
+		container.style.maxHeight = transformedHeight;
+	}
+
+	private _setContentsDomNodeMaxDimensions(width: number | string, height: number | string): void {
 		const contentsDomNode = this._hover.contentsDomNode;
-		contentsDomNode.style.maxWidth = transformedWidth;
-		contentsDomNode.style.maxHeight = transformedHeight;
+		ContentHoverWidget._applyMaxDimensions(contentsDomNode, width, height);
+		console.log('contentsDomNode : ', contentsDomNode);
```
Comment (alexr00) [inline src/vs/editor/contrib/hover/browser/contentHover.ts]: And here too.

**Review by alexr00 — CHANGES_REQUESTED** [review]

**Review by alexr00 — DISMISSED** [review]
And now they're gone ✅

**Review by alexdima — COMMENTED** [review]
Seems to work well! I've left a few comments.

**Review by alexdima — APPROVED** [review]

**Comment by alexdima** [comment]
Please hold off from merging until tomorrow.

**Pull request merged**